aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -575,26 +595,12 @@ impl AptosVM {
         payload: &EntryFunction,
         new_published_modules_loaded: &mut bool,
     ) -> Result<(), VMStatus> {
-        let function =
-            session.load_function(payload.module(), payload.function(), payload.ty_args())?;
-        // This transaction is now being executed as the multisig account.
```

Comment:
Can we keep this comment here?

Reply:
This flow is exactly the same as normal tx, what is the extra information?

Reply:
That whole comment is extra information, such as 

// If txn args are not valid, we'd still consider the multisig transaction as executed but
// failed. This is primarily because it's unrecoverable at this point.

Reply:
I added the comments to the validate_and_execute_entry_function function where they are in logically the same place

Reply:
But these comments are specific to the multisig execution?

Reply:
Why are they specific to multisig? Normal tx also treat invalid args as executed but failed